zebra-grpc/src/server.rs

```diff
@@ -0,0 +1,47 @@
+//! The gRPC server implementation
+
+use tonic::{transport::Server, Response, Status};
+
+use scanner::scanner_server::{Scanner, ScannerServer};
+use scanner::{Empty, InfoReply};
+
+use zebra_scan::{storage::Storage, Config};
+
+/// The generated scanner proto
+pub mod scanner {
+    tonic::include_proto!("scanner");
+}
+
+#[derive(Debug, Default)]
+/// The server implementation
+pub struct ScannerRPC {}
+
+#[tonic::async_trait]
+impl Scanner for ScannerRPC {
+    async fn get_info(
+        &self,
+        _request: tonic::Request<Empty>,
+    ) -> Result<Response<InfoReply>, Status> {
+        let network = zcash_primitives::consensus::Network::MainNetwork;
+        let storage = Storage::new(&Config::default(), network.into(), true);
```

Comment:
We could access `Storage` through a service if we reverse the dependency.

Reply:
A service sounds good but i can't see the benefit apart from more work. If the service is inside `zebra-scan` we might need to depend on it anyway? Can you explain more the rationality of the suggested change ?

What i understand is that we will eventually want the grpc server to get started when zebra starts. If it is the grpc calling the scanner or the reverse is the same to me right now but i will like to see what i am not seeing here and make a team decision.

I will be happy to make the change if that is what we decide to do.

Reply:
Ok, fair enough, thanks. I think we can do this with little changes to this PR. I made the mods at https://github.com/ZcashFoundation/zebra/pull/8178/commits/92c1b7d731f20701e633dcdf128a818cf4a63381